app/api/models/statistics.py

```diff
@@ -26,15 +26,43 @@ class Coding(BaseModel):
     average_intron_length: float = None
```

Comment:
Sorry, missed in Mehrnaz's PR. Corrections for the coding stats above:

- `shortest_gene_length` should be int rather than float
- `longest_gene_length` — same
- `coding_transcripts` — same